sssom/parsers.py

```diff
@@ -589,7 +590,9 @@ def from_obographs(
                                     )
                                     mdict["subject_label"] = label
                                     mdict["predicate_id"] = "oboInOwl:hasDbXref"
-                                    mdict["match_type"] = MATCH_TYPE_UNSPECIFIED
+                                    mdict[
+                                        "mapping_justification"
```

Comment:
can we start using constants for this, like in constants.py 

MAPPING_JUSTIFICATION="mapping_justification" and then

mdict[MAPPING_JUSTIFICATION]=MAPPING_JUSTFN_UNSPECIFIED

Reply:
you also do this in util.py, so you can just move all the constants in there to constants.py and then import them in all other files?